### DISCUSSION TITLE
Jollyany | Multi-purpose Joomla Template

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#734] [Q&A]
I bought the Jollyany | Multi-purpose Joomla template from Envato about 2 years ago but have never used it. Today I wanted to install it on my local development environment and see how things are done there. The installation and activation basically worked, but that was it. After installing a template with the corresponding extensions, I only get error messages when calling the frontend.

See screenshot.

![tp](https://github.com/user-attachments/assets/b940d21b-056d-483f-b5bd-ec53d2adc277)

Hat jemand eine Idee warum?

**Comment by sonvnn** [comment]
Can you give me more information? 

- PHP Version
- Joomla Version
- Local Apps

Thank you!

**Comment by Banuk1616** [comment]
@sonvnn

I have tried to install it under the following local development environment:
Bearsampp_2024.6.27
Joomla 5.1.2 also testet with Joomla 5.0.1
PHP 8.3.7 also tested with PHP 8.2.19
MariaDB 11.4.2 also tested with MariaDB 10.9.7
Jollyany 3.5.28

For this I first set up a new local host. Then I installed Joomla and then installed the template_package.zip file in the Joomla backend. Until then everything worked fine, even the activation, but as soon as I call the frontend, I get this error message. 

![jollyany1](https://github.com/user-attachments/assets/958cb9dc-adad-44e8-90c0-9fdacc5db586)

Because I thought it might be related to my local development environment Bearsammp, I also tested this with Xampp 8.2.12, but got the same error message.

This problem is not so important for me now and there is no hurry. I bought Jollyany about 2 years ago but never installed it. Now I wanted to install it locally to see how one or the other is done. Sometimes I can learn better if I can see how something is done.

Best regards Olli

Translated with DeepL.com (free version)
